### ISSUE TITLE
Java API is missing ability to enable stemming

### ISSUE THREAD

**Issue by stevemcleod** [typesense/typesense-java#68]
## Description

According to the Typesense docs, there's a `stem` option .

The `org.typesense.model.Field` class is missing the `stem` option.

## Steps to reproduce

No repro steps.

I'm using the latest release of the Java Typesense API, as follows:

```
        <dependency>
            <groupId>org.typesense</groupId>
            <artifactId>typesense-java</artifactId>
            <version>0.9.0</version>
        </dependency>
```

I'd be happy to submit a PR, but the `org.typesense.model.Field` seems to be auto-generated from an `openapi.yml` file, which isn't part of this repo (https://github.com/typesense/typesense-java/).

**Comment by jasonbosco** [comment]
Looks like we missed adding that in. 

Thank you for offering to do a PR. The OpenAPI specs are in this repo: https://github.com/typesense/typesense-api-spec

**Comment by stevemcleod** [comment]
> Thank you for offering to do a PR.

I was a bit unclear. I don't think I can do a PR.

If this issue required only some Java coding, then I would be able to a PR. But the `org.typesense.model.Field` class I would edit doesn't actually exist in the repo. So I guess it is created at build time?

I think it is a build configuration problem, in that the Typesense Java API is being built with the wrong version of the Swagger config file, and hence the generated classes are missing the `stem` setting.

I'm not certain, because I'm unfamiliar with both Swagger and Gradle.

Sorry I can't help more with this.

**Comment by jasonbosco** [comment]
Ah ok no worries. 

CC: @phiHero

**Comment by phiHero** [comment]
Hi @stevemcleod

>But the org.typesense.model.Field class I would edit doesn't actually exist in the repo. So I guess it is created at build time?

Yes the schemas are generated at build time using Swagger. 

I have added stemming in the  typesense-api-spec so it should be included in the next release of typesense-java.

**Comment by kishorenc** [comment]
Fixed in `v1.0.0` release.
